lib/todo/console_display.ex

```diff
@@ -2,16 +2,18 @@ defmodule ToDo.ConsoleDisplay do
   import ToDo.MenuOptionsFormatter
   import ToDo.TasksFormatter
 
+  @prompt "> "
+
   def display_to_do_list_and_cheatsheet(tasks) do
     display_put(tasks_for_display(tasks) <> menu_options_for_display)
   end
 
   def prompt_for_command do
-    display_get(">")
+    String.downcase(String.strip(display_get(@prompt)))
   end
 
   def display_get(message) do
-    IO.gets(message)
+    IO.gets message
```

Comment:
I would keep the parentheses here. It's best to pick a style and be consistent with it (no paren vs paren). Personally I prefer parentheses everywhere as it makes things more explicit.